go/chat/sender.go

```diff
@@ -845,6 +845,10 @@ func (s *Deliverer) failMessage(ctx context.Context, obr chat1.OutboxRecord,
 				s.outbox.GetUID(), err.Error())
 			return err
 		}
+	case chat1.OutboxErrorType_ALREADY_DELETED:
```

Comment:
what's the reason we don't do this for `OutboxErrorType_DUPLICATE`?

Reply:
I guess I just didn't care. This is fine too, but maybe also bring duplicate in too? mayb can combine those two switches.

Reply:
sgtm